showcase.proto

```diff
@@ -22,43 +22,44 @@ import "google/rpc/status.proto";
 
 package google.showcase.v1;
 
+
 // A service to showcase Generated API Client features and common API patterns
 // used by Google.
 service Showcase {
   // A method used to show unary methods. This method will return the message
   // that was given.
-  rpc Echo(EchoRequest) returns (EchoResponse) {}
+  rpc Echo(EchoRequest) returns (EchoResponse);
```

Comment:
Sanity check: I assume you intentionally skipped having `google.api.http` annotations since you do not (at least not yet anyway) have an HTTP server equivalent?